live/tests/integration/components/qcu-proposals-test.js

```diff
@@ -35,10 +35,10 @@ describe('Integration | Component | QcuProposals', function() {
       this.set('answerChanged', answerChangedHandler);
 
       // when
-      this.render(hbs`{{qcu-proposals answers=answers proposals=proposals onAnswerUpdated='answerChanged'}}`);
+      this.render(hbs`{{qcu-proposals answers=answers proposals=proposals answerChanged='answerChanged'}}`);
```

Comment:
on aurait pu. mais c'était plus simple dans ce cas de l'écrire comme ça. sinon il aurait fallu utiliser la méthode `this.on('action')` dans un contexte de test.